### DISCUSSION TITLE
Will Panon continue in background if I play games through Wine/Steam?

### DISCUSSION THREAD

**Discussion by aliquo93** [rbn42/panon#57] [Q&A] [answered]
Hi, if I play a game using Wine or Steam, will Panon continue its animations while it's in background or it automatically stops?  Because, if so, it would be consuming extra resources.

Thanks in advance for your help!

**Comment by rbn42** [comment, marked as answer]
> will Panon continue its animations while it's in background

Sadly yes.

**Reply by aliquo93** [reply]
> > will Panon continue its animations while it's in background
> 
> Sadly yes.

I suspected it, because I have an old PC and immediately noticed a decrease in games performance... well I hope you could fix it in newer releases! Anyway, congratulations for the awesome work, it's a great widget!
